tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+# tasks file for ansible-role-bes-sonarQube
+
+- include: "tasks/oah-install.yml"
+  when: oah_command == "install"
+
+- include: "tasks/oah-remove.yml"
+  when: oah_command == "uninstall"
```

Comment:
command should be remove

Reply:
Change is made already [ uninstall -> remove ]